### PR TITLE
Nhse d34 otp26

### PR DESCRIPTION
**Pull request by martinsumner**
Add folsom in specifically to riak_repl (as now removed from exometer library used elsewhere in riak)

### CODE REVIEW

File: rebar.config [hunk 1]
```diff
@@ -22,8 +22,9 @@
 
 {deps, [
         {ranch, {git, "https://github.com/ninenines/ranch.git", {tag, "1.8.0"}}},
-        {ebloom,{git, "https://github.com/nhs-riak/ebloom.git", {branch, "nhse-develop"}}},
-        {riak_kv, {git, "https://github.com/nhs-riak/riak_kv.git", {branch, "nhse-develop"}}}
+        {ebloom, {git, "https://github.com/nhs-riak/ebloom.git", {branch, "nhse-develop"}}},
```
Comment (ThomasArts) [inline rebar.config:25]: is that ebloom version more in line than ebloom branch develop-3.0?

File: rebar.config [hunk 2]
```diff
@@ -22,8 +22,9 @@
...
+        {folsom, {git, "https://github.com/nhs-riak/folsom.git", {branch, "nhse-develop-3.4"}}},
+        {riak_kv, {git, "https://github.com/nhs-riak/riak_kv.git", {branch, "nhse-d34-otp26"}}}
```
Comment (ThomasArts) [inline rebar.config]: possibly a temporary branch to point to?

**Review by ThomasArts — COMMENTED** [review]
Not sure about the branch choices.

File: rebar.config (hunk 2 above)
Comment (martinsumner) [reply rebar.config]: Now references updated permanent branch

**Pull request merged**